activejob/test/cases/railtie_test.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "helper"
+require "active_support/testing/isolation"
+
+class RailtieTest < ActiveSupport::TestCase
+  include ActiveSupport::Testing::Isolation
+
+  setup do
+    require "rails"
+
+    rails_logger = Logger.new(nil)
+
+    @app ||= Object.const_set(
+      "AJRailtieTestApp#{SecureRandom.alphanumeric(6)}",
+      Class.new(::Rails::Application)
+    ) do
```

Comment:
```suggestion
    @app ||= Class.new(::Rails::Application) do
      def self.name; "AJRailtieTestApp"; end
```

The application needs to have a `name`, but it doesn't need to be assigned to a constant.